app/models/invitation.rb

```diff
@@ -3,16 +3,10 @@ class Invitation < ApplicationRecord
   has_many :users, through: :invitation_users
 
   after_initialize :gen_code
-  before_validation :set_current
 
   private
 
   def gen_code
-    self.code ||= SecureRandom.hex(4)
-    self.limit ||= 1
-  end
-
-  def set_current
-    self.current ||= self.limit
+    self.code ||= SecureRandom.hex(4) # TODO: Make this safer since codes are unique
```

Comment:
waaaaaaaaaaay to long for an invite code, imo, although I did consider it.

Reply:
It's also not a super big problem right now, as the form just returns an error saying the code isn't unique, since this is an `after_initialize` callback.